lib/templates/middleAssignment/Screen2.dart

```diff
@@ -0,0 +1,92 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class Screen2 extends StatefulWidget {
+  @override
+  _Screen2State createState() => _Screen2State();
+}
+
+class _Screen2State extends State<Screen2> {
+  late Future<List<Session>> _sessionsFuture;
+
```

Comment:
Why it?

<img width="359" alt="Screenshot 2024-06-24 at 20 51 54" src="https://github.com/timur-harin/Sum24FlutterCourse/assets/64702874/6a002646-5517-4da9-bcf5-8d0b15d0f7cb">